shared/react-native/ios/Keybase/AppDelegate.m

```diff
@@ -43,6 +45,19 @@ - (BOOL)addSkipBackupAttributeToItemAtPath:(NSString *) filePathString
   return success;
 }
 
+- (void) createBkgReadableDir:(NSString*) path
```

Comment:
I found this when I was working on this:

https://forums.developer.apple.com/thread/24483

Seems to also apply to stuff that is already in there (or at least that is how it played out for me).